example/README.md

```diff
@@ -28,7 +28,7 @@ Steps
 
 Steps
 1. Build the SDK
-2. In the Example `package.json` change the `@openfga/sdk` dependency from a semver range like below
+2. In the Example `package.json` change the `@openfga/sdk` dependency from a server range like below
```

Comment:
Semver is correct here as it's referring to the version in the dependencies stanze

```suggestion
2. In the Example `package.json` change the `@openfga/sdk` dependency from a semver range like below
```

Reply:
Ohh okay